Jenkinsfile

```
@@ -106,6 +106,15 @@ node('java') {
             // If no project is selected - build all of them.
             // This functionality is important in order to allow triggering by SCM.
             def projectsToBuild = selectedProjects ?: projectsConfig.keySet()
+
+            // Build build-info first
+            if (projectsToBuild.contains("build-info")) {
+                stage("Building build-info") {
+                    def buildConfig = projectsConfig["build-info"]
+                    buildProject(deployServer, rtGradle, buildConfig.buildName, buildConfig.buildTasks)
+                    projectsToBuild -= "build-info"
```

```suggestion
                selectedProjects.remove("build-info")
```
This is more readable.